bootstrap.py

```diff
@@ -707,6 +708,7 @@ def install_ssh_key_from_string(foreman_ssh_key):
             print_generic("Foreman's SSH key already present in %s" % options.remote_exec_authpath)
             return
     output = os.fdopen(os.open(options.remote_exec_authpath, os.O_WRONLY | os.O_CREAT, OWNER_ONLY_FILE), 'a')
+    output.write("\n")
     output.write(foreman_ssh_key)
```

Comment:
Would this now need a newline at the end?

Reply:
Nah, the file will just not have an ending `\n`, which I think is fine.

Reply:
I think I've seen some tools fail on that so perhaps better to be safe than sorry?

Reply:
It's already likely the user has a line ending so you might be injecting additional newlines anyway. But I'm fine either way.